docs/actions.rst

```diff
@@ -51,6 +44,9 @@ All actions in Crud, even your own, should inherit from the
 This class is ``abstract`` and provides numerous auxiliary methods which can be
 useful for you both as a developer as an action creator.
 
+Where you choose to put your custom Crud actions is up to you, but using ``src/Crud`` is a good place. Remember to
```

Comment:
Your explanation makes total sense to me. Since it was in the docs before I imagine @jippi has already thought about it. The only thing I want and think is useful... one single recommended namespace (`src/Crud`).